psi4/driver/aliases.py

```diff
@@ -85,7 +82,7 @@ def fake_file11(wfn, filename='fake_file11.dat', **kwargs):
                 '', gradient.get(at, 0), gradient.get(at, 1), gradient.get(at, 2)))
 
 
-def sherrill_gold_standard(func, label, **kwargs):
+def sherrill_gold_standard(**kwargs):
     r"""Function to call the quantum chemical method known as 'Gold Standard'
```

Comment:
true. docstrings have always been awkward with the wrappers since there isn't usually a centralized function. I may have a strategy, but it's not immediate. papered over the trouble for now, I think.